crds/v1alpha1/src/main/crds/applicationsets.argoproj.io

```diff
@@ -15307,7 +15307,7 @@ spec:
                     step:
                       type: string
                   required:
-                  - application
+                  - applicationConfig
```

Comment:
This definitely doesn't seem right. I don't think that CRDs changed since last time I pulled them.